### PR TITLE
Update 'How to run' section of README.md

### PR DESCRIPTION
**Pull request by fernanda-freitas**
Hello there!

What I addressed in this Pull Request:

- Fixed the typo in the 'Running Backend Node JS Application'
- Added a new step to create the .env file and to paste content from .env.example inside
- Added markdown quotes to make the commands easier to read.

Thanks ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,9 +49,10 @@ If you prefer the command line way of downloading and installing things, then fe
   ##### `git clone https://github.com/lugenx/ecohabit.git`
 - Open the cloned folder in VS Code. Open the terminal and make sure its pointing to the root of the cloned project.
 
-- Running Backend Nods JS Application:
+- Running Backend Node JS Application:
 
-  - Change the directory to server folder using terminal command: cd server
+  - Change the directory to server folder using terminal command: `cd server`
+  - In the server directory, create an .env file and copy the content from .env.example into the new .env file.
```
Comment (tsimian) [inline README.md:55]: Could you please reword this to 'At the root of the server directory'? This way it explicitly states _where_ in the server folder.

**Review by tsimian — APPROVED** [review]
I left a comment for a minor change. Other than that it looks good to me @lugenx! Thanks for your help @fernanda-freitas!

**Comment by lugenx** [comment]
Thank you! 👍
